src/utils/helpers/list/mapColumnsByRole.ts

```diff
@@ -29,7 +29,7 @@ import { ColumnDetails } from '../../types'
 
 const allColumns: { [key in APPLICATION_COLUMNS]: ColumnDetails } = {
   SERIAL_NUMBER: {
-    headerName: '',
+    headerName: 'Serial',
```

Comment:
i think this is headerless in designs ?

Reply:
Yes, I thought those design headers were kind of "provisional". 

I dont' really mind either way. In one sense it looks better without it, but if you sort by that column, the sort arrow without a word next to it looks kind of silly, so... 🤷‍♂️